src/main/java/com/auth0/client/mgmt/ResourceServerEntity.java

```diff
@@ -0,0 +1,95 @@
+package com.auth0.client.mgmt;
+
+import com.auth0.client.mgmt.builder.RequestBuilder;
+import com.auth0.json.mgmt.ResourceServer;
+import com.auth0.net.Request;
+import com.auth0.utils.Asserts;
+import com.fasterxml.jackson.core.type.TypeReference;
+import okhttp3.HttpUrl;
+import okhttp3.OkHttpClient;
+
+import java.util.List;
+
+public class ResourceServerEntity  {
+    private final RequestBuilder requestBuilder;
+
+    ResourceServerEntity(OkHttpClient client, HttpUrl baseUrl, String apiToken) {
+        requestBuilder = new RequestBuilder(client, baseUrl, apiToken);
+    }
+
+    /**
+     * Creates request to fetch all resource servers.
+     * See <a href=https://auth0.com/docs/api/management/v2#!/Resource_Servers/get_resource_servers>API documentation</a>
+     *
+     * @return request to execute
+     */
+    public Request<List<ResourceServer>> list() {
+
+        return requestBuilder.get("api/v2/resource-servers")
+                             .request(new TypeReference<List<ResourceServer>>() {
+                             });
+    }
+
+    /**
+     * Cretes request for fetching single resource server by it's ID.
+     * See <a href=https://auth0.com/docs/api/management/v2#!/Resource_Servers/get_resource_servers_by_id>API documentation</a>
+     *
+     * @param id {@link ResourceServer#identifier} field
```

Comment:
`id` and `identifier` are different properties of a ResourceServer. My guess is that the `id` is the one the server assigns when it's created, and the identifier is the name we choose for it (like "myapp.auth0.com"). An `identifier` is required when creating the ResourceServer while the `id` is always returned when listing/getting resource servers.

Reply:
thanks, fixed